packages/slate-react/src/components/inline.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react'
+import { getRelativeRange } from './block'
 
+import Text from './text'
 import DATA_ATTRS from '../constants/data-attributes'
 
 export default class Inline extends React.Component {
+  tmp = { nodeRefs: {} }
```

Comment:
Just curious mostly, did you find you _need_ this, or just copying it because block.js had it too?

Reply:
Just copied it lol